sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -1974,7 +1974,7 @@ def from_dict(cls, data: dict) -> "TextAppearance":
 class BoundingRegion:
     """The bounding region corresponding to a page.
 
-    :ivar Optional[Sequence[~azure.ai.formrecognizer.Point]] polygon:
+    :ivar Sequence[~azure.ai.formrecognizer.Point] polygon:
```

Comment:
This is marked as required in the swagger. Adding it here to get it merged today.

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/2fd7dcd89afa70ff5ba7be88bee987da62099a28/specification/cognitiveservices/data-plane/FormRecognizer/stable/2022-08-31/FormRecognizer.json#L2076